layout/includes/comments/vssue.pug

```diff
@@ -0,0 +1,35 @@
+doctype html
```

Comment:
??? unnecessary, don't add too many unnecessary codes.

Reply:
It's a component, not a page, don't add these, just follow other comment components.